File: csharp-dotnet2/src/main/CsharpDotNet2/estuary-client/Model/TypesIpfsListPinStatusResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace estuary-client.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class TypesIpfsListPinStatusResponse {
    /// <summary>
    /// Gets or Sets Count
    /// </summary>
    [DataMember(Name="count", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "count")]
    public int? Count { get; set; }

    /// <summary>
    /// Gets or Sets Results
    /// </summary>
    [DataMember(Name="results", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "results")]
    public List<TypesIpfsPinStatusResponse> Results { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class TypesIpfsListPinStatusResponse {\n");
      sb.Append("  Count: ").Append(Count).Append("\n");
      sb.Append("  Results: ").Append(Results).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: dart/lib/model/types_ipfs_list_pin_status_response.dart
part of swagger.api;

class TypesIpfsListPinStatusResponse {
  
  int count = null;

  List<TypesIpfsPinStatusResponse> results = [];

  TypesIpfsListPinStatusResponse();

  @override
  String toString() {
    return 'TypesIpfsListPinStatusResponse[count=$count, results=$results, ]';
  }

  TypesIpfsListPinStatusResponse.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    count = json['count'];
    results = TypesIpfsPinStatusResponse.listFromJson(json['results']);
  }

  Map<String, dynamic> toJson() {
    return {
      'count': count,
      'results': results
     };
  }

  static List<TypesIpfsListPinStatusResponse> listFromJson(List<dynamic> json) {
    return json == null ? new List<TypesIpfsListPinStatusResponse>() : json.map((value) => new TypesIpfsListPinStatusResponse.fromJson(value)).toList();
  }

  static Map<String, TypesIpfsListPinStatusResponse> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, TypesIpfsListPinStatusResponse>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new TypesIpfsListPinStatusResponse.fromJson(value));
    }
    return map;
  }
}
